recipes/ncl/yum_requirements.txt

```diff
@@ -0,0 +1,8 @@
+csh
+imake
+libx11-devel
+libXaw-devel
+libXmu-devel
+byacc
+flex
+flex-devel
```

Comment:
Can you try the `flex` we have in `conda-forge`? If that does not work you can add only `flex-devel` as that should install `flex` too.